connect/runtime/src/test/java/org/apache/kafka/connect/storage/OffsetStorageWriterTest.java

```diff
@@ -192,6 +209,236 @@ public void testCancelAfterAwaitFlush() throws Exception {
         flushFuture.get(1000, TimeUnit.MILLISECONDS);
     }
 
+    @Test
+    public void testFlushFailureWhenWriteToSecondaryStoreFailsForTombstoneOffsets() throws InterruptedException, TimeoutException, ExecutionException {
```

Comment:
```suggestion
    public void testFlushFailureWhenWriteToSecondaryStoreFailsForTombstoneOffsets() throws Exception {
```
nit: since this is just a test method, we can avoid the verbosity

Reply:
Done.

Reply:
Removed such instances from other tests as well.